f3/granite.go

```diff
@@ -517,37 +515,31 @@ func (i *instance) beginConverge() {
 	if signers, signatures, ok := prevRoundState.committed.FindStrongQuorumAgreement(ZeroTipSetID()); ok {
 		value := ECChain{}
 		aggSignature := i.host.Aggregate(signatures, nil)
-		justificationPayload := SignedMessage{
+		justificationPayload := Payload{
```

Comment:
Nit: I suggest inlining this into the `Justification{ ... }` constructor here and everywhere else that it isn't yet too.